### ISSUE TITLE
step() masked from base, but nonfunctional

### ISSUE THREAD

**Issue by ichneumonid** [bbolker/glmmadmb#8]
the step() function for stepwise deletion is masked, but it is not actually functional, at least in the glmmADMB package I had (and reloaded9/15/17 or so, to make sure it was up to date)

**Comment by bbolker** [comment]
a **brief** response: the problem is that `step()` is not a generic function (so it's hard to modify how it behaves for a particular model type), but was misbehaving in some bad way (TBH I no longer remember how/why) for glmmADMB fits.  

Test out the fix in 332c179 (i.e. `devtools::install_github("bbolker/glmmadmb")` and reload) and see if it seems like an adequate solution?

**Comment by ichneumonid** [comment]
step() wasn't working for glmmADMB fits, it said it wasn't able to (I don't
recall exactly what it said), but it also masked the same function from the
base, so I couldn't use it elsewhere in the code where it would ordinarily
work.

I can't actually check that this fixed it, as I can't get the package to
reload, despite using the same code I used to load it initially:

install.packages("glmmADMB",
    repos=c("http://glmmadmb.r-forge.r-project.org/repos",
            getOption("repos")),
    type="source")
library(glmmADMB)

It gives me errors now:

ERROR: compilation failed for package 'plyr'
* removing 'C:/Users/lopre/OneDrive/Documents/R/win-library/3.4/plyr'
ERROR: dependency 'stringi' is not available for package 'stringr'
* removing 'C:/Users/lopre/OneDrive/Documents/R/win-library/3.4/stringr'
ERROR: dependencies 'stringr', 'plyr' are not available for package
'glmmADMB'
* removing 'C:/Users/lopre/OneDrive/Documents/R/win-library/3.4/glmmADMB'

(And a whole bunch of warnings because of the non-loaded subpackages).

So, in short, despite trying repeatedly, I can't tell if this fixed
anything.

-Eric
